Bottom: 87ea387015ce932aee6447e9749e74914de1f19b
Top:    6fbdd1fa922fa0be40367d905008b40048aace0a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:16:36 +0000

Fix make distcheck


---

diff --git a/w32/Makefile.am b/w32/Makefile.am
index af86e32..5355f61 100644
--- a/w32/Makefile.am
+++ b/w32/Makefile.am
@@ -2,8 +2,6 @@ EXTRA_DIST= build-all.sh minipack.conf mpk README \
   patches/gd patches/gd/0001-Export-sysbols-when-building-as-a-DLL.patch \
   patches/gd/0002-Libtool-requires-no-undefined-to-build-a-DLL.patch \
   patches/gd/0003-Avoid-conflit-between-libjpeg-and-windows-headers.patch \
-  patches/gettext/01-revert-sed-string.patch \
-  patches/gettext/02-Avoid-missing-open-argument-error.patch \
   patches/gtk+ patches/gtk+/01-mousewheel.patch \
   patches/zlib patches/zlib/01-shared-lib-support.patch \
   patches/zlib/02-cross-build.patch \
